2-mandatory/4-restaurant.js

```diff
@@ -20,7 +20,26 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
+  orderBurger: function (balance) {
+    let burgerPrice = balance - MENU.burger;
```

Comment:
Not too important but we could use more descriptive variables so instead of balance could use remainingBalance